### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

**Comment by tribal-tec** [comment]
Missing AUTHORS and ACKNOWLEDGEMENTS files

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Servus - A small set of network oriented utilities in C++ including a zeroconf implementation.
+
+Servus is licensed under the LGPL version 3, unless noted otherwise, e.g., for external dependencies. See file COPYING.txt for the full license. External dependencies are either LGPL or BSD-licensed. See file ACKNOWLEDGEMENTS.txt and AUTHORS.txt for further details.
```
Comment (eile) [inline LICENSE.txt]: Lunchbox is LGPL V2, so this is relicensing. From a quick check, only @biddisco and @delyas are external and need to confirm. Guys - are you fine with this?

File: doc/RelNotes.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+### Version 0.1
+
+* Initial version.
+
+- - -
+
+# About {#About}
+
+Servus is partially cross-platform, the only mandatory dependency is using
+a C++11 compiler. Zeroconf will be available in those platforms were either
```
Comment (eile) [inline doc/RelNotes.md]: I would not make C++11 mandatory here, since it will affect all our projects.

File: tests/uint128_t.cpp [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+//            std::ostringstream stream;
+//            stream << uuid;
+//            uuid = stream.str();
+//            BOOST_CHECK_EQUAL( uuid, j->first );
+//
+//            // BOOST_CHECK_EQUAL fails to compile
+//            BOOST_CHECK( first.find( uuid ) == first.end( ));
+//            first[ uuid ] = true;
+//        }
+//    }
```
Comment (eile) [inline tests/uint128_t.cpp]: ?

File: doc/RelNotes.md (hunk 2 above)
Comment (eile) [inline doc/RelNotes.md]: Nevermind, the (rng) implementation seems to need it. As discussed, we'll #ifdef with boost if the need for C++03 arises.

File: LICENSE.txt (hunk 1 above)
Comment (hernando) [inline LICENSE.txt]: I kept the original copyright notice of every file, so I don't think this is a concern.
Regarding the authors I have found @eile, Cedric, @tribal-tec and @bilgili, but no one else. John has contributed to the CMake subtree, but that should be considered a separate repo, shouldn't it.

Comment (eile) [inline LICENSE.txt]: I would prefer to relicense all files to be clear. Seems nothing major from the two "externals":

https://github.com/Eyescale/Lunchbox/commit/99b209025395bf32c66c1b8dcabc40c342079185#diff-0e416b135e5619b56a01648e287548e3
https://github.com/Eyescale/Lunchbox/commit/0875c026cfd65d9657cbc61b7ed8bf4a4e9fb653

Comment (delyas) [inline LICENSE.txt]: Works for us.

**Comment by hernando** [comment]
I pushed to origin by mistake

**Pull request closed without merging**